### DISCUSSION TITLE
Site Monitor and Alert

### DISCUSSION THREAD

**Discussion by c9mb** [umbraco/Umbraco.Cloud.Issues#892] [Ideas]
When deploying multiple sites for clients (particularly using baseline deployment) monitoring the status of those sites quickly becomes an issue.

There are 3rd-party commercial services available to monitor sites and send email/sms alerts when down, but it's awkward to monitor the status of the environment, database, and project, and it would be more convenient if monitoring/alerts could be configured and managed as part of the Umbraco-Cloud Portal settings.

This could perhaps be offered as a reasonably/competitively priced add-on and incorporated into the monthly fee.

**Comment by KennethJakobsen** [comment]
Can you offer a bit more detail into what you want, is it on deployment?

**Comment by azure-devops-sync** [comment]
AB#643

**Comment by c9mb** [comment]
In essence, when you deploy sites to Umbraco-Cloud you take it 'on faith' that the site will remain running and publicly available.

Unfortunately, such faith would be misplaced, and just like any website hosting, you cannot assume 100% reliability and uptime.

I recently had a situation with Umbraco-Cloud where a couple of my sites became unresponsive, and I had no way of knowing it with the default tools.

You could employ a 3rd-party commercial solution to 'ping' the site and alert the admin if it becomes unresponsive - e.g. https://www.siteuptime.com/ - but that only tells you the basic website response status, not the specifics of the issue e.g. not the status of the azure-sql service.

Given that Umbraco-Cloud is running on Azure, I assumed that there would be more extensive status information already available to the cloud devs, that would allow them to provide a service config to alert the admin if the public site is unresponsive, or the sql service is hung, or if other critical conditions require the admin's attention.

**Comment by nul800sebastiaan** [comment]
Hi @c9mbundy, going through some old issue reports. We are in the process of moving to an improved Cloud platform and that is going to be our main focus for the foreseeable future.

This feature request is really good but unfortunately not something we can offer on our current platform. Once we have completed the move to the new platform we are in a much better position to offer a feature like this and we'll get back to it at that time.

**Comment by pijemcolu** [comment]
Hostname monitoring is a feature we offer for a while now: https://docs.umbraco.com/umbraco-cloud/optimize-and-maintain-your-site/monitor-and-troubleshoot/hostname-monitoring

Alerting is something we're currently working on. I'll close this issue, you can follow-up here: https://github.com/umbraco/Umbraco.Cloud.Issues/discussions/1042
